tests/framework/console/FakeController.php

```diff
@@ -84,6 +84,10 @@ public function actionAksi6()
         return $this->testArray;
     }
 
+    public function actionAksi7(self $typedArgument, $simpleArgument) {
```

Comment:
can you rename this action towards a name that is less magical and closer related to the actual test scenario?